auth-middleware-test-svc/requirements.txt

```diff
@@ -1,5 +1,6 @@
 Flask==1.1.1
 gunicorn==19.9.0
 connexion[swagger-ui]==2.4.0
+python-dotenv==0.10.3
```

Comment:
It looks like dotenv is not used in the code.

Reply:
added dotenv line and tested

Reply:
Thanks.